concepticondata/conceptlists/Vergallito-2020-1121.tsv

```diff
@@ -438,7 +438,7 @@ Vergallito-2020-1121-436	436	gioia	joy	2524	JOY	5	n	0.9666666667	545.0344828	1	5
 Vergallito-2020-1121-437	437	gioiello	jewel	1029	JEWEL	8	n	0.9666666667	623.5172414	1	544.1034483	-0.1076989198	-0.02677638422	0.7	0.12	4.26	0.32	4.81	v	45.93535749	4.81	0.12	2.042	6.472287076
 Vergallito-2020-1121-438	438	gioioso	joyful			7	a	1	526.8	1	563.4615385	-0.6601568649	-0.08105291657	3.95	0.6	0.88	0.58	4.21	v	35.518591	4.21	0.58	2.044	5.898932107
 Vergallito-2020-1121-439	439	giovane	young	1207	YOUNG	7	a	0.9666666667	566.7241379	1	524.9285714	-0.4648435141	-0.2200217587	3.14	0.33	1.95	0.91	4.49	v	38.44731978	4.49	0.33	2.164	5.895693343
-Vergallito-2020-1121-440	440	gioventù	youth			8	n	1	606.5333333	1	496.4230769	-0.1706705624	-0.5387421225	3.31	0.25	1.29	0.77	4.29	v	40.76690212	4.29	0.25	1.982	5.628472262
+Vergallito-2020-1121-440	440	gioventù	youth	1304	CHILD (YOUNG HUMAN)	8	n	1	606.5333333	1	496.4230769	-0.1706705624	-0.5387421225	3.31	0.25	1.29	0.77	4.29	v	40.76690212	4.29	0.25	1.982	5.628472262
```

Comment:
This is a decendent of Latin "iuventud", so it is the abstract concept of "youth", not the "young person".